manifests/params.pp

```diff
@@ -310,13 +316,23 @@
   }
 
   $server_jvm_java_bin = '/usr/bin/java'
-  $server_jvm_min_heap_size = '2G'
-  $server_jvm_max_heap_size = '2G'
+  $server_jvm_min_heap_size = '4G'
+  $server_jvm_max_heap_size = '4G'
```

Comment:
is this needed for puppetserver 2? If not, please leave the 2G, I was already stumped when I switched to puppetserver 1 for module testing and my VM got maxed out.